flink-runtime/src/main/java/org/apache/flink/runtime/deployment/ResultPartitionDeploymentDescriptor.java

```diff
@@ -103,6 +137,9 @@ public boolean sendScheduleOrUpdateConsumersMessage() {
 	 * by {@link ShuffleMaster#releasePartitionExternally(ShuffleDescriptor)}
 	 * and {@link ShuffleEnvironment#releasePartitionsLocally(Collection)}.
 	 *
+	 * <p>The partition has to support the corresponding {@link ReleaseType} in
```

Comment:
I think it might be better to indicate which value true/false should support which `releaseType`, otherwise it seems not clear the mapping between this value and release type.